assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_uploadAll.vm

```diff
@@ -100,7 +100,7 @@
 						onclick="ASN.invokeDownloadUrl('$accessPointUrl', '$actionString', '$alertMessage', '$contextString', '$viewString', '$searchString', '$showSubmissionByFilterSearchOnly', this);" value="$tlang.getString('downloadall.button.download')" />
 			#else
 				<input type="button" name="uploadButton" id="uploadButton"  accesskey="s" class="active"
-						onclick="SPNR.disableControlsAndSpin( this, null ); document.getElementById('uploadAllForm').action='#toolLinkParam($action 'doUpload_all')'; ASN.submitForm( 'uploadAllForm', null, null, null );" value="$tlang.getString('uploadall.button.upload')" />
+						onclick="SPNR.disableControlsAndSpin( this, null ); document.getElementById('uploadAllForm').action='#toolLinkParam($action "doUpload_all" "sakai_csrf_token=$validator.escapeUrl($sakai_csrf_token)")'; ASN.submitForm( 'uploadAllForm', null, null, null );" value="$tlang.getString('uploadall.button.upload')" />
```

Comment:
The quotes don't seem consistent here?

Reply:
I do not know why, but it does not work with single quotes